analysis/parser/RawTextParser/src/test/kotlin/de/maibornwolff/codecharta/rawtextparser/MetricCollectorTest.kt

```diff
@@ -33,18 +33,57 @@ class MetricCollectorTest {
     }
 
     @Test
-    fun `Should exlude regex patterns`() {
+    fun `Should exclude regex patterns`() {
         val result = MetricCollector(File("src/test/resources/sampleproject").absoluteFile, exclude = arrayOf(".*\\.xyz", "foobar")).parse()
 
-        Assertions.assertThat(result.size).isEqualTo(1)
+        Assertions.assertThat(result.size).isEqualTo(2)
         Assertions.assertThat(result).containsKey("/spaces/spaces_xyz.wrong")
     }
 
     @Test
-    fun `Should include only spedified File extensions`() {
+    fun `Should include only specified File extension with one given`() {
         val result = MetricCollector(File("src/test/resources/sampleproject").absoluteFile, fileExtensions = arrayOf("wrong")).parse()
 
         Assertions.assertThat(result.size).isEqualTo(1)
         Assertions.assertThat(result).containsKey("/spaces/spaces_xyz.wrong")
+        Assertions.assertThat(result).doesNotContainKey("/spaces/spaces_3.xyz")
+        Assertions.assertThat(result).doesNotContainKey("tabs.xyz")
+    }
+
+    @Test
+    fun `Should include only specified File extensions with multiple given`() {
+        val result = MetricCollector(File("src/test/resources/sampleproject").absoluteFile, fileExtensions = arrayOf("wrong", "xyz")).parse()
+
+        Assertions.assertThat(result.size).isEqualTo(4)
+        Assertions.assertThat(result).containsKey("/spaces/spaces_xyz.wrong")
```

Comment:
"wrong" confuses me since the result contains this file. 
Let's improve the namings here a bit to indicate what is excluded and what not (.excluded, .included).